files/ptn0/scripts/prereqs.sh

```diff
@@ -58,9 +58,9 @@ echo "Installing Cabal 3.0.0 .."
 wget https://downloads.haskell.org/cabal/cabal-install-3.0.0.0/cabal-install-3.0.0.0-x86_64-unknown-linux.tar.xz
 tar xf cabal-install-3.0.0.0-x86_64-unknown-linux.tar.xz
 chmod 755 cabal
-mkdir -p ~/.cabal/bin
 mv cabal ~/.ghcup/bin
 rm -f cabal-install-3.0.0.0-x86_64-unknown-linux.tar.xz cabal.sig
+export PATH=$PATH:~/.ghcup/bin
```

Comment:
~/.ghcup/bin should be added to $PATH via adding of ~/.ghcup/env in source (part of ghcup install), tho I havnt tested it since adding the silent mode installation (the interactive mode does prompt for confirmation to add ghcup/env to profile).

However, I agree we might want to reference the env file within script (to ensure the existing session retains the PATH reference too)

Reply:
I did test it from a debian-minimal image and PATH as is (not interactively) was not updating it. we should add ~/.cabal/bin as well as stated earlier.

Reply:
Try installing ghcup without the environment variable added recently.

Reply:
I think that:
```
. ~/.ghcup/env L50
```
has to replace the export after the cabal install and that will do the trick, since both cabal and ghup path are included there.. we may want also to do something like:
```
echo "source ~/.ghcup/env" >> ~/.bashrc
```
??
-- Note: I prefer "source" vs "." because I like it more :)